cmd/horcrux/cmd/config.go

```diff
@@ -0,0 +1,249 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/jackzampolin/horcrux/internal/signer"
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+func init() {
+	// TODO: config nodes add/remove
+	// TODO: config peers add/remove
+	// TODO: config chain-id set
+	configCmd.AddCommand(initCmd())
+	rootCmd.AddCommand(configCmd)
+}
+
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Commands to configure the horcrux signer",
+}
+
+func initCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "init [chain-id] [chain-nodes]",
+		Aliases: []string{"i"},
+		Short:   "initalize configuration file and home directory if one doesn't already exist",
+		Long: "initalize configuration file, use flags for cosigner configuration.\n\n" +
+			"[chain-id] is the chain id of the chain to validate\n" +
+			"[chain-nodes] is a comma seperated array of chain node addresses i.e.\n" +
+			"tcp://chain-node-1:1234,tcp://chain-node-2:1234",
+		Args: cobra.RangeArgs(1, 2),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			cid := args[0]
```

Comment:
FYI; This feels like it could panic with `runtime error: index out of range [0]` if a user runs it w/o providing an argument. (Maybe there's upstream protection on this tho)

Reply:
The range args function will prevent this